device/CMakeLists.txt

```diff
@@ -41,16 +41,15 @@ target_include_directories(
 )
 
 # flatbuffers is public - exposed to tt_metal by tt_simulation_device_generated.h
-# nng is public - exposed to tt_metal by tt_simulation_host.hpp
```

Comment:
@vtangTT  Does this have some implication for simulator usage in tt_metal?

Reply:
yep should be fixed I forgot to add comment